vispy/visuals/volume.py

```diff
@@ -628,7 +614,7 @@ def _create_vertex_data(self):
         
         # Get indices and vertices for triangles. 
         indices, tex_coord, ver_coord = self._calc_coords(tex_coord, ver_coord,
-                                                          div)
+                                                          2)
```

Comment:
I suppose you can remove lots of lines above in this function. Which is awesome, it should even increase performance a wee bit.